Core/src/game/functions/booleans/is/line/IsLine.java

```diff
@@ -218,8 +254,26 @@ public boolean eval(final Context context)
 	
 			final TopologyElement vertexLoc = playOnCell ? graph.cells().get(locn) : graph.vertices().get(locn);
 			final ContainerState state = context.state().containerStates()[context.containerId()[vertexLoc.index()]];
-			final TIntArrayList whats = new TIntArrayList();
+			final TIntArrayList whats = new TIntArrayList(); 
 			final int whatLocn = state.what(locn, type);
+			
+			if (isVisibleFn.eval(context) == true) {
+				if ((locn == 8 && state.what(9, SiteType.Vertex) != 0) || (locn == 10 && state.what(11, SiteType.Vertex) != 0) || (locn == 14 && state.what(15, SiteType.Vertex) != 0) || (locn == 18 && state.what(19, SiteType.Vertex) != 0) || (locn == 20 && state.what(21, SiteType.Vertex) != 0)) {
```

Comment:
what are all the magic numbers (8, 9, 10, 11, 14, 15, 18, 19) here? 

Is this hardcoded for the Shibumi board? If so, at minimum, need some comment here to say that. But maybe it'd be even better to have some safety check somewhere, to print an error early if this ludeme is being used while the Shibumi board isn't. Not 100% sure where the best place for that would be though. 

@epiette thoughts?